lisk-dex-electron/src/App.css

```diff
@@ -153,9 +153,13 @@ li {
 
 .modal-foreground {
   position: absolute;
-  z-index: 3;
+  display: flex;
+  justify-content: center;
+  align-content: center;
+  width: 100vw;
```

Comment:
`vw` units are dangerous because of an old bug in certain iOS devices which causes them to get messed up when you switch from landscape to portrait mode and back. Let me know if this bug is no longer relevant.

Reply:
I'll fix this with percentages tomorrow

Reply:
@jondubois changed it to percentages